pkg/users/account-details.js

```diff
@@ -182,11 +181,9 @@ export function AccountDetails({ accounts, groups, shadow, current_user, user })
 
     if (!accounts.length) {
         return (
-            <EmptyState variant={EmptyStateVariant.small}>
-                <Spinner isSVG size="xl" />
-                <Title headingLevel="h1" size="lg">
-                    {_("Loading...")}
-                </Title>
+            <EmptyState variant={EmptyStateVariant.sm}>
+                <EmptyStateHeader titleText={_("Loading...")} headingLevel="h1" />
+                <EmptyStateFooter><Spinner size="xl" /></EmptyStateFooter>
```

Comment:
These 3 added lines are not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-085951-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)